.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
```

Comment:
I guess this is gonna be pretty controversial topic 😂 , but can we have a `printWidth` of 100? I kinda prefer having things in one line rather than splint over n lines, and higher printWidth helps that:

example from `example/index.js` - how to turn 3 lines to like 13:

```js
import { AppRegistry, StyleSheet, Text, View, Alert, Button } from 'react-native';

import { GoogleSignin, GoogleSigninButton, statusCodes } from 'react-native-google-signin';
```
=>
```
import {
  AppRegistry,
  StyleSheet,
  Text,
  View,
  Alert,
  Button,
} from 'react-native';

 import {
  GoogleSignin,
  GoogleSigninButton,
  statusCodes,
} from 'react-native-google-signin';
```

imports are an extreme case that does not matter too much, but it illustrates the point.

Reply:
I'm not convinced but whatever xD